api-gateway/src/main/resources/static/scripts/auth/forgot-pwd-form/forgot-pwd-form.controller.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+angular.module('forgotPwdForm')
+    .controller('forgotPwdFormController', ["$http", '$location', "$scope", function ($http, $location, $scope) {
```

Comment:
usually ive seen in the project the controller names to be all capital but im not sure